packages/cli/package.json

```diff
@@ -75,14 +75,13 @@
 		"@smartthings/core-sdk": "^4.2.0",
 		"@smartthings/plugin-cli-edge": "^1.14.1",
 		"aws-sdk": "^2.1144.0",
-		"cli-table": "^0.3.11",
 		"inquirer": "^8.2.4",
 		"js-yaml": "^4.1.0",
-		"log4js": "6.3.0"
+		"log4js": "6.3.0",
+		"table": "^6.8.0"
```

Comment:
It doesn't look like this is actually required for the cli package.

Reply:
Good catch. Fixed.